woocommerce/woocommerce
# Add hightlighter to the tree control

**Depends on #36590**


### All Submissions:






### Changes proposed in this Pull Request:



Depends on https://github.com/woocommerce/woocommerce/pull/36435
Partialy closes https://github.com/woocommerce/woocommerce/issues/35851






### How to test the changes in this Pull Request:



1. Run storybook
2. Visit `http://localhost:6007/?path=/story/woocommerce-admin-experimental-treecontrol--highlight-first-matching-item`
3. After typing the first matching item should be highlighted



### Other information:




### FOR PR REVIEWER ONLY:
